index.js

```diff
@@ -5,20 +5,27 @@ const async = require('async');
 const defaults = require('lodash.defaultsdeep');
 
 class PageData {
-  constructor(url, key) {
+  constructor(url, key, userAgent) {
     this.options = {
```

Comment:
you could just set useragent in options, if it's not passed through, nothing it set